### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ionas**
... make it similar to FoC/cakephp-csvview

### CODE REVIEW

**Comment by ionas** [comment]
Ref: https://github.com/FriendsOfCake/cakephp-csvview/pull/59

File: README.md [hunk 1]
```diff
@@ -12,22 +12,24 @@ Search provides a search module for CakePHP applications.
 The master branch has the following requirements:
 
 * CakePHP 3.0.0 or greater.
+* Patience
```
Comment (ADmad) [inline README.md]: There is no need for this.

File: README.md [hunk 2]
```diff
@@ -12,22 +12,24 @@ Search provides a search module for CakePHP applications.
...
 
 ## Installation
 
 * Install the plugin with composer from your CakePHP Project's ROOT directory
 (where composer.json file is located)
-```sh
-php composer.phar require friendsofcake/search
+```shell
+php composer.phar require friendsofcake/search:~1.2
```
Comment (ADmad) [inline README.md]: There's no need to specify version. Composer will automatically use latest release and set version constraints in composer.json

File: README.md (hunk 1 above)
Comment (bravo-kernel) [inline README.md]: :flushed:

**Comment by ionas** [comment]
In case you are happy with this, I will do the same for CsvView.

**Comment by bravo-kernel** [comment]
I know it's just a matter of taste but I would suggest using the installation order as used in e.g. https://github.com/usemuffin/slug as that uses the IMHO preferable order to load the plugin (console first, manual otherwise). All to put new users onto the right path from the get go.

## Installation

Using [Composer](https://getcomposer.org)

```bash
composer require muffin/slug:~1.0
```

To make your application load the plugin either run:

```bash
./bin/cake plugin load Muffin/Slug
```

or add the following line to `config/bootstrap.php`:

```php
Plugin::load('Muffin/Slug');
```

**Comment by davidyell** [comment]
I don't understand what benefits this brings, if any at all.

**Pull request closed without merging**